src/ralph/backends/mixins.py

```diff
@@ -0,0 +1,39 @@
+"""Backend mixins for Ralph"""
+
+import json
+import logging
+
+from ralph.defaults import HISTORY_FILE
+
+logger = logging.getLogger(__name__)
+
+
+class HistoryMixin:
```

Comment:
Do we need to a way to clean history without knowing where the file is?

Reply:
Good idea!

Reply:
Done.